### ISSUE TITLE
Windows installer creates passive nrdp.cfg file with commands that are different to example.cfg

### ISSUE THREAD

**Issue by box293** [NagiosEnterprises/ncpa#396]
This does not affect the functionality of NCPA however I think it's important that what is in the documentation (https://www.nagios.org/ncpa/help/2.0/passive.html#configuring) and examples should be used by the installer. It avoids any confusion for the end user.

Here is example.cfg (I've removed the #'s):
```
%HOSTNAME%|__HOST__ = system/agent_version
%HOSTNAME%|CPU Usage = cpu/percent --warning 60 --critical 80 --aggregate avg
%HOSTNAME%|Swap Usage = memory/swap --warning 60 --critical 80 --units Gi
%HOSTNAME%|Memory Usage = memory/virtual --warning 80 --critical 90 --units Gi
%HOSTNAME%|Process Count = processes --warning 300 --critical 400
```

Here is what the installer presents and uses in the `nrdp.cfg` file it creates:

![2017-10-17 05 passive checks](https://user-images.githubusercontent.com/8356113/31749251-8c7bf392-b4c4-11e7-8802-a67cc87402cb.png)

The differences are:
 - All of the api endpoints begin with `/` whereas in `example.cfg` they don't
 - `--units Gi` is not used on the Swap Usage and Memory Usage checks

**Comment by jomann09** [comment]
f3ad5e17634aa335c74fc6c300d9f3f3cbf7f93c

**Comment by box293** [comment]
It looks like the checks still begin with a `/`.

In `example.cfg` the new Disk Usage check begins with `/`

**Comment by jomann09** [comment]
I fixed this in cb3f8bc050b0004eb0e773c4e33649373c78eb5e
